library/ssl_cookie.c

```diff
@@ -167,7 +167,9 @@ int mbedtls_ssl_cookie_write( void *p_ctx,
     if( (size_t)( end - *p ) < COOKIE_LEN )
         return( MBEDTLS_ERR_SSL_BUFFER_TOO_SMALL );
 
-#if defined(MBEDTLS_HAVE_TIME)
+#if defined(FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION)
+    t = 0x5af2a056;
```

Comment:
Why this specific value?

Reply:
No reason. But we need a constant value so that fuzzing results are reproducible even if we explicitly want to avoid this in TLS.